pkg/translator/prometheusremotewrite/helper.go

```diff
@@ -536,3 +536,29 @@ func addResourceTargetInfo(resource pcommon.Resource, settings Settings, timesta
 func convertTimeStamp(timestamp pcommon.Timestamp) int64 {
 	return timestamp.AsTime().UnixNano() / (int64(time.Millisecond) / int64(time.Nanosecond))
 }
+
+
+func translatorMetricFromOtelMetric(metric pmetric.Metric) otlptranslator.Metric {
```

Comment:
One alternative to duplicating this code every where, is using the one in Prometheus that was already exported. What do you folks think?

https://github.com/prometheus/prometheus/blob/f1c6fab7e15b7328dd35a3aa46a468d8a0f100a7/storage/remote/otlptranslator/prometheusremotewrite/metrics_to_prw.go#L74

Reply:
In general, the purpose of otlptranslator is to remove duplication between the two codebases, so I would say yes.  Is there a reason why we haven't moved it already?

Reply:
@dashpole , I made this change but now `googlemanagerprometheusexporter` depends on prometheus/prometheus, please let me know if that ok or if you'd like me to revert just for google's exporter

Reply:
That is OK with me.  Thanks for making the update!